--- /usr/portage/dev-libs/nss/nss-3.12.10.ebuild	2011-05-13 23:13:30.000000000 +0200
+++ /usr/local/portage/dev-libs/nss/nss-3.12.10.ebuild	2011-07-12 17:55:42.040725579 +0200
@@ -51,6 +51,10 @@
 		lib/ssl/config.mk || die
 	sed -i -e "/CRYPTOLIB/s:\$(SOFTOKEN_LIB_DIR):../../lib/freebl/\$(OBJDIR):" \
 		cmd/platlibs.mk || die
+
+	# copy Linux2.6.mk to Linux3.0.mk if the latter doesn't exist
+	test -f "${S}"/mozilla/security/coreconf/Linux3.0.mk ||
+		cp "${S}"/mozilla/security/coreconf/Linux2.6.mk "${S}"/mozilla/security/coreconf/Linux3.0.mk
 }
 
 src_compile() {
